plugins/user-settings/src/components/General/UserSettingsProfileCard.tsx

```diff
@@ -13,18 +13,25 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Grid, Typography } from '@material-ui/core';
+import { Grid, makeStyles, Typography } from '@material-ui/core';
 import React from 'react';
 import { UserSettingsSignInAvatar } from './UserSettingsSignInAvatar';
 import { UserSettingsMenu } from './UserSettingsMenu';
 import { useUserProfile } from '../useUserProfileInfo';
 import { InfoCard } from '@backstage/core-components';
 
+const useStyles = makeStyles({
+  fullWidth: {
+    width: '100%',
+  },
+});
+
 export const UserSettingsProfileCard = () => {
   const { profile, displayName } = useUserProfile();
+  const classes = useStyles();
 
   return (
-    <InfoCard title="Profile">
+    <InfoCard title="Profile" className={classes.fullWidth}>
```

Comment:
But confused about the `width` here, are they not already stretched horizontally? 😅

Regarding the height alignment, I believe we want to add the `gridItem` variant to the `InfoCard`s rather than applying custom styles